docs/core/index.md

```diff
@@ -7,137 +7,41 @@ ms.date: 06/20/2016
 ---
 # .NET Core Guide
 
-> Check out the ["Getting Started" tutorials](get-started.md) to learn how to create a simple .NET Core application. It only takes a few minutes to get your first app up and running.
+[.NET Core](about.md) is an [open source](https://github.com/dotnet/coreclr/blob/master/LICENSE.TXT) general purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It is cross-platform, supporting Windows, macOS and Linux, and can be used in device, cloud, and IoT applications.
 
-.NET Core is a general purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It is cross-platform, supporting Windows, macOS and Linux, and can be used in device, cloud, and embedded/IoT scenarios. 
+See [About .NET Core](about.md) to learn more about .NET Core, including its characteristics, supported languages and frameworks, and key APIs.
 
-The following characteristics best define .NET Core:
+Check out [.NET Core Tutorials](tutorials/index.md) to learn how to create a simple .NET Core application. It only takes a few minutes to get your first app up and running. If you want to try something really quick, you can [try .NET Core in your browser](https://docs.microsoft.com/dotnet/csharp/quick-starts/hello-world).
```

Comment:
the first app is actually shown on get started. should we send them there first instead?

If you want to try..., you can try -> a bit repetitive, also usually Acrolinx will complain about things like really quick